bsp/freedom-e310-arty/metal-platform.h

```diff
@@ -125,4 +125,6 @@
 #define METAL_SIFIVE_UART0_IP 20UL
 #define METAL_SIFIVE_UART0_DIV 24UL
 
+#define METAL_SIFIVE_HCA
```

Comment:
Why are these get added if the feature does not appears in the dts file?

Reply:
look like an issue with the freedom-devicetree-tools generation. I should probably remove it, because this define is not used ;)